server/graphql/v1/queries.js

```diff
@@ -859,6 +863,12 @@ const queries = {
         include: [],
       };
 
+      if (args.renderCollectivesFromList) {
+        query.where.id = await Promise.all(
+          args.renderCollectivesFromList.map(async id => (typeof id === 'string' ? await fetchCollectiveId(id) : id)),
+        );
```

Comment:
Should just be a where `IN` condition with the slugs. As simple as it can be.

Reply:
Yes. I just made that change.